tests/test-email-prepare.php

```diff
@@ -0,0 +1,94 @@
+<?php
+include_once dirname( __FILE__ ) . '/email-test-case.php';
+
+/**
+ * Class testing Caldera_Forms_Email_Prepare
+ */
+class Test_Email_Prepared extends Email_Test_Case {
+
+
+	/**
+	 * test return email from rfc822
+	 */
+	public function test_email_from_rfc822() {
+		$_rfc      = 'Roy Sivan <roy@sivan.com>';
+		$_expected = 'roy@sivan.com';
```

Comment:
@robwilde Please do not use `$_var_name`. It signifies a "private" variable to me. But now I'm wondering why, code readability is decreased as a result.

Reply:
Will update